Gemfile

```diff
@@ -29,6 +29,7 @@ gem 'rack-cors', '~> 2.0'
 gem 'sass-rails', '~> 6.0.0'
 gem 'sendgrid', '~> 1.2.4'
 gem 'sprockets', '~> 4.2.1'
+gem 'strong_migrations', '~> 1.6', '>= 1.6.3'
```

Comment:
Shouldn't it be available only for development?

Reply:
Yeah, otherwise we should update the Readme description so it doesn't specify "in development"

Reply:
I think the gem is needed in all envs by design. 

The "in development" I think comes from the gem's README that says:
>Catch unsafe migrations in development

I think the point is that if you write an unsafe migration you'll catch it during development because you won't be able to even run it locally. Not sure what happens if you try to push to prod, I guess it breaks the build

Reply:
It not only reads migrations but also inspects the schema. It makes sense that it's not limited solely to the development environment, as it could also detect errors if a production push is done.